newshour-namedentity/readme.md

```diff
@@ -1 +1,87 @@
+Options include `person, organization, program_title, publication_title, product, location, & event`.  
+* _**Character Start Offset** & **Character End Offset** - These two numbers entail which chars in the text file span the named entity's text. 
+This information is **automatically added** to the raw dataset file by the brat tool. 
+e.g. "Jim Lehrer" spans from character 32 inclusive to character 42 exclusive, taking up 10 letters. The first character of a file is 0._    
+* _(Note, Wikipedia link is not used in this project. It's used in the [next project](https://github.com/clamsproject/aapb-annotations/tree/main/newshour-namedentity-wikipedialink).)_ 
+### How to Annotate It
+(See Above)
+### Decisions, Differentiation, and Precision Level during Annotation
+* **Category Disambiguation** - While this process is relatively straightforward, there are times when it can be difficult to place an item under one category umbrella. 
+For example, "Washington" can refer to the person, US capital, the organization within it, or the northwestern state. 
+A similar issue can arise with colleges, many of whom are named after the state or city they're located in. In these cases, context is key. 
+A relatively reliable rule of thumb is, if a location is mentioned, but no politicians or professors are brought up, then file it under location. Otherwise, file it under organization.  
+
+* **Transcript Match** - Annotation Leads and Project Managers should ensure the transcripts accurately reflect the video that they are supposed to match before annotators begin.  
+Please also consider whether the transcripts and labelling should label only spoken text or also extraneous script information such as the name of who is speaking, non-spoken information, etc. 
+
+* **General Typos** - As with any labelling with text, there is a possibility of typos. Data Quality checks such as for typos have not been done on these datasets.  
+
+## Data Format and `process.py`
+### `raw` data
+`.ann` file where each line is a new instance of a named entity. Information on the `.ann` [file format](https://brat.nlplab.org/standoff.html) is here. 
+* Fields:
+    * `category` - which type of named entity the text is. 
+    * `first_character` - first char that starts the text string within the transcript, inclusive. The first character of a file is 0.
+    * `last_character` - last char of the text string in the transcript, exclusive. 
+    * `entity_text` - what the text of the named entity is. 
+* Format:
+```
+tag#\tcategory first_character last_character\tentity_text
+```
+* Example:
+```
+T1	person 2 12	JIM LEHRER
+T2	person 32 42	Jim Lehrer
+T3	program_title 51 59	NewsHour
+```
+### [`process.py`](process.py)
+_TODO (discrepancy): It seems that the `process.py` and all other `.py` files used to transform from raw to gold is no longer needed. If the process here is no longer necessary, it should be deleted and moved to version control or named to denote previous versioning._  
```

Comment:
I think we still use the current `process.py` to copy raw files into golds directory; 
https://github.com/clamsproject/aapb-annotations/blob/2240945a823b75a077c9e5427498d63a69498a1a/newshour-namedentity/process.py#L142-L143

So we can 
1. keep other parts of the file and the dependency as is (it will hurt the readability of directory, but nothing major)
2. trim the `process.py`, leaving the copy part only and deletion of other `.py` files in the same commit for future trackability. 

Either way, I'm happy. If you think the second way is cleaner, I can help adding that commit.

Reply:
Ok. I'll write a summary saying that the process.py does that, yes, that the process.py does the copying. 
However, it becomes more difficult to know for sure that I'm providing an accurate depiction for the other files and the total sum of the code in this directory.

I think that 1. is the better option, in case its ever needed, and the readme can explain that they are not used historically. 
Would definitely appreciate your hand in writing something to best explain this! =)

Reply:
An explanation of the process.py simple usage was added.